### ISSUE TITLE
What's wrong with it?

### ISSUE THREAD

**Issue by dameer** [diegoversiani/smooth-parallax#3]
HTML structure:
```
...
<div class="site-offset">
    <a href="#" id="k-flux" smooth-parallax="" container=".site-offset" start-movement="0.05">FLUX</a>
    <div id="content" class="content">
    ...
    </div>
</div>
...
```

Init as required:
```
<script type="text/javascript">
  window.addEventListener("load", function () {
    SmoothParallax.init();
  });
</script>
```

If well understood the point, #k-flux should move within ".site-offset". No error, warning or whatsoever. It just doesn't work.

**Comment by diegoversiani** [comment]
Hi Damer,

There is one missing required attribute on your element:

> Configure elements movement
> You'll also have to set at least one more attribute `start-position` or `end-position`, see options at standard options.

Please check again de sections [Basic Usage](https://github.com/diegoversiani/smooth-parallax#basic-usage) and [Standard Options](https://github.com/diegoversiani/smooth-parallax#standard-options).

Sorry for the documentation is not as good as we'd like, I added a task to my personal to-do list to fix it.

Best!

**Comment by dameer** [comment]
Yeah, got it workin now. It was because I kinda thought that if no position is set, the plugin will pick up defaults. Overlooked required config tho.
Thanks for your work mate, I like the plugin!

**Comment by diegoversiani** [comment]
That's good :)

I would expect it too, I think when I made it my needs where a bit different and having the required config made sense.

I'll keep this issue open until I fix the documentation.
